src/components/common-components/Notifications.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <ul id="messages" v-for="message in messages" :key="message.id">
+        <li>{{message.message}}</li>
+    </ul>
+</template>
+
+<script lang="ts">
+import { defineComponent, getCurrentInstance, inject } from 'vue'
+import {ActionTypes} from '@/store/modules/auth/actions'
+import {mapActions, mapGetters} from 'vuex'
+
+
+export default defineComponent({
+    computed:{
+        ...mapGetters({
+            user: 'getUser',
+            user_company: 'getUserCompany',
```

Comment:
User company is already fetched in userdata.

Reply:
changed